cfme/tests/cloud_infra_common/test_cloud_init_provisioning.py

```diff
@@ -28,6 +30,17 @@
 ]
 
 
+def find_global_ipv6(vm):
+    """
+    Find global IPv6 on a VM if present and return it as a string. Otherwise, return False.
+    """
+    all_ips = vm.mgmt.all_ips
+    for ip in all_ips:
+        if ':' in ip and not ip.startswith('fe80'):
+            return ip
+    return False
```

Comment:
I would recommend to return None to be consistent with wrapanapi.

Reply:
@digitronik I tried it after @rhrazdil suggested it. Problem with that is that if this function returns `None` if case of failure, `wait_for` considers it finished and does not perform another round.